cmd/respawn/respawn.go

```diff
@@ -69,6 +69,12 @@ func run(c *cli.Context) error {
 		panic(err)
 	}
 
+	// Now that we're booted, stop writing debug messages to the console
```

Comment:
Maybe we should just always turn this off in `console_init` and make it configurable later - having it available on first boot is the most important thing anyways.